sqlalchemy_history/manager.py

```diff
@@ -446,54 +446,15 @@ def track_association_operations(self, conn, cursor, statement, parameters, cont
             op = Operation.DELETE
 
         if op is not None:
-            table_name = statement.split(" ")[2]
+            table_name = (
```

Comment:
Is there a util function we can use from sqlalchemy to do this

Reply:
This is not required as this table_name is used in the metadata.tables mapping
not for sql query generation